DTM/Hills_DTM/map.json

```diff
@@ -0,0 +1,106 @@
+{
+	"name": "Hills",
+	"authors": ["Clijmart, MilkyWayRules"],
+	"version": "1.0",
+	"gametype": "DTM",
+	"teams": [
+		{
+			"id": "blue",
+			"name": "Blue",
+			"color": "blue",
+			"max": 50,
+			"min": 1
+		},
+		{
+			"id": "red",
+			"name": "Red",
+			"color": "red",
+			"max": 50,
+			"min": 1
+		}
+	],
+	"spawns": [
+		{"teams": ["spectators"], "coords": "0, 100, 0"},
+		{"teams": ["Blue"], "coords": "83.5,99,77.5,135,0"},
```

Comment:
The coordinates for Red and Blue's spawn doesn't have a space after the comma, that's a bit annoying.